app/src/main/res/values/strings.xml

```diff
@@ -516,14 +516,14 @@
     <string name="website">Website</string>
     <string name="website_message" translatable="false">https://muntashirakon.github.io/AppManager</string>
     <string name="pref_remove_all_rules">Remove all rules</string>
-    <string name="pref_remove_all_rules_msg">All configured blocked components will be enabled, app ops will be set to default values and permissions will be granted</string>
+    <string name="pref_remove_all_rules_msg">No components will be blocked, app ops will be set to default values and permissions granted.</string>
```

Comment:
The change doesn't seem to reflect the original meaning (the original meaning is also somewhat incorrect though). There are three ways a component can be configured: enable, disable and default (block/unblock terminologies are widely used in privacy communities though). When a component is unblocked, it isn't enabled by AM rather it is set to default (Android figures out what to do with the component). But AM only unblocks the components that are _configured_ within AM. If you block a component using MAT, for example, but not on AM, the component won't be enabled.